src/index.js

```diff
@@ -0,0 +1,3 @@
+import './style.css';
```

Comment:
Please follow the wireframe for the navigation menu. The logo should be on the left side of the navigation items.

Home page footer text should be on the left side of as shown in the mock-up.